python_modules/dagster/dagster/_cli/project.py

```diff
@@ -32,6 +32,8 @@ def project_cli():
     "This CLI enables you to quickly bootstrap your project with an officially maintained example."
 )
 
+list_examples_command_help_text = "List the examples that available to bootstrap with."
+
```

Comment:
that _are_ available

Reply:
Maybe shorten the entire thing to "List available examples for the from-example command."